src/vconsole/vconsole-setup.c

```diff
@@ -140,6 +142,9 @@ static int keyboard_load_and_wait(const char *vc, const char *map, const char *m
                 args[i++] = map_toggle;
         args[i++] = NULL;
 
+        log_debug("Executing \"%s\"...",
+                  strnull((cmd = strv_join((char**) args, " "))));
```

Comment:
not a big fan of doing so much in a single expression. i mean, it doesn't really hurt doing this in two lines i think. but ok